### PR TITLE
[ElmSharp] Fix Transit.Go to start transition immediately

### PR DESCRIPTION
**Pull request by jaehyun0cho**
Transit.Go calls elm_transit_go_in with interval.
elm_transit_go_in uses timer so it does not start transition immediately
although interval is 0.

To start transition immediately without interval, Transit.Go is fixed to
call elm_transit_go.

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/ElmSharp/ElmSharp/Transit.cs [hunk 1]
```diff
@@ -246,7 +246,18 @@ public double EndAccelerationFactor
         /// <since_tizen> preview </since_tizen>
         public void Go(double interval = 0)
         {
-            Interop.Elementary.elm_transit_go_in(_handle, interval);
+            Double timeInterval = interval;
+
+            if (timeInterval.Equals(0.0))
```
Comment (myroot) [inline src/ElmSharp/ElmSharp/Transit.cs:251]:
```suggestion
            if (timeInterval == 0)
```

File: src/ElmSharp/ElmSharp/Transit.cs [hunk 2]
```diff
@@ -246,7 +246,18 @@ public double EndAccelerationFactor
         /// <since_tizen> preview </since_tizen>
         public void Go(double interval = 0)
         {
-            Interop.Elementary.elm_transit_go_in(_handle, interval);
+            Double timeInterval = interval;
```
Comment (myroot) [inline src/ElmSharp/ElmSharp/Transit.cs:249]:
```suggestion
```

File: src/ElmSharp/ElmSharp/Transit.cs (hunk 1 above)
Comment (jaehyun0cho) [reply src/ElmSharp/ElmSharp/Transit.cs]: Thank you for review. I have updated the patch.

File: src/ElmSharp/ElmSharp/Transit.cs (hunk 2 above)
Comment (jaehyun0cho) [reply src/ElmSharp/ElmSharp/Transit.cs]: Thank you for review. I have updated the patch.

**Review by myroot — APPROVED** [review]

**Comment by rookiejava** [comment]
One simple question. I couldn't find any information about that on [API reference](https://docs.tizen.org/application/native/api/mobile/5.5/group__Elm__Transit.html#ga5f39f8f0c030eacd601c9d9c6c89ebee).

>elm_transit_go_in uses timer so it does not start transition immediately although interval is 0.

Is this behavior a change from Tizen 6.0 M2? I'm curious because the existing version (4.0, 5.0, 5.5 and even 6.0 M1) works well with the current code.

**Review by rookiejava — APPROVED** [review]

**Pull request merged**